model/ftn/w3fld1md.ftn

```diff
@@ -1091,9 +1091,11 @@
       !----------------------------------------------
       DO K=KA1, KA2-1
          AVG=SUM(INSPC(K,:))/MAX(REAL(NTH),1.)
-         DO T=1,NTH
-            INSPC(K,T)=BT(K)*INSPC(K,T)/TPI/(WN2(K)**3.0)/AVG
-         ENDDO
+         IF (AVG .GT. 0.0) THEN
```

Comment:
There is a slightly different coding of this fix in the WW3 main.

Reply:
thanks for noticing @dabail10 - I will double check

Reply:
thans Dave - fixed.